### DISCUSSION TITLE
Volume issues vs. previous build?

### DISCUSSION THREAD

**Discussion by LarryXO** [PlummersSoftwareLLC/NightDriverStrip#353] [Q&A]
I recently made a second Insulators set up, and used the web installer. I have it sitting next to my old one, and it seems like the new build requires 2x the volume of the music before it reacts. Similarly, on the Spectrum setup, the new web builds seem to have issues with lower volumes the previous ones don't. Has anyone else experienced this?

**Comment by davepl** [comment]
We made some changes recently to fine tune the processing for the MESMERIZER mic set.  That might upset other projects that use different mics.

Try changing your MIN_VU (double it, or half it, see which way helps) and if that helps, let me know and we can put a conditional in.

Basically, MAX_VU (to a lesser extent) and MIN_VU define the active range and it’s a little different for every mic and ever attenuation level.

- Dave

**Reply by LarryXO** [reply]
I altered line 90 in soundanalyzer.h to make MINVU 128 instead of 256, but after compiling and loading, there seemed to be no difference. I will test further.
